sdk/formrecognizer/ai-form-recognizer/README.md

```diff
@@ -135,29 +157,39 @@ async function main() {
   await poller.pollUntilDone();
   const response = poller.getResult();
 
-  console.log(`### Response status ${response.status}`);
-  const usReceipt = response.receipts[0];
-  console.log("First receipt:")
-  console.log(`Receipt type: ${usReceipt.receiptType}`)
-  console.log(`Merchant Name: ${usReceipt.merchantName.value} (confidence: ${usReceipt.merchantName.confidence})`);
-  console.log(`Transaction Date: ${usReceipt.transactionDate.value} (confidence: ${usReceipt.transactionDate.confidence})`);
-  console.log("Receipt items:");
-  console.log(`  name\tprice\tquantity\ttotalPrice`);
-  for (const item of usReceipt.items) {
-    const name = `${optionalToString(item.name?.value)} (confidence: ${optionalToString(item.name?.confidence)})`;
-    const price = `${optionalToString(item.price?.value)} (confidence: ${optionalToString(item.price?.confidence)})`;
-    const quantity = `${optionalToString(item.quantity?.value)} (confidence: ${optionalToString(item.quantity?.confidence)})`;
-    const totalPrice = `${optionalToString(item.totalPrice?.value)} (confidence: ${optionalToString(item.totalPrice?.confidence)})`;
-    console.log(`  ${name}\t${price}\t${quantity}\t${totalPrice}`);
+  if (!receipts || receipts.length <= 0) {
+    throw new Error("Expecting at lease one receipt in analysis result");
   }
 
-  // raw fields are also included in the result
-  console.log("Raw 'MerchantAddress' field:");
-  console.log(usReceipt.recognizedForm.fields["MerchantAddress"]);
-}
-
-function optionalToString(value) {
-  return `${value || "<missing>"}`;
+  const receipt = receipts[0];
```

Comment:
I liked the approach that Python had in the Readme of just iterating through the fields and then in the samples show some of the fields that are included.
Feel free to change it or leave as is :)

Reply:
I'd like to only maintain one sample to make easier to copy working code into the docs.